typescript/introduction/intro-typescript/what-is-typescript.md

```diff
@@ -0,0 +1,40 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Intro to TypeScript
+
+---
+## Content
+
+Depending on how old you are, you might remember the time JavaScript was just a web development language. Due to its increased versatility, JavaScript is now used in many applications such as designing neural networks, power backends and many others.
+
+You're probably thinking 'Hey, I thought this was a TypeScript course!' yet here we are talking about JavaScript. Because of the high adoption rate, many frameworks (Angular) and libraries (React) were created that aid with development. 
+
+TypeScript is another creation meant to put the future of JavaScript in the hands of developers, and it actually is a superset of JavaScript. What it means is that TypeScript retains all the functionality and adds more.
```

Comment:
This is a very important feature of TypeScript. In fact, the #1 reason for its popularity is probably this interoperability with JavaScript. Converting js to ts is done by just changing the file extension.

```suggestion
TypeScript is another creation meant to put the future of JavaScript in the hands of developers, and it actually is a superset of JavaScript. What this means is that TypeScript retains all the functionality and adds more.
```